### PR TITLE
printing of cpv2 finalization committee parameters

### PR DESCRIPTION
**Pull request by EmilLa1**
## Purpose

Addresses https://github.com/Concordium/concordium-node/issues/740

## Changes

Printing of the `FinalizationCommitteeParameters` for cpv2. 

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: src/Concordium/Client/Output.hs [hunk 1]
```diff
@@ -1144,6 +1144,12 @@ printChainParametersV2 ChainParameters {..} = tell [
   [i|     * timeout decrease: #{showRatio (_cpConsensusParameters ^. cpTimeoutParameters ^. tpTimeoutDecrease)}|],
   [i|  + minimum time between blocks: #{_cpConsensusParameters ^. cpMinBlockTime}|],
   [i|  + block energy limit: #{_cpConsensusParameters ^. cpBlockEnergyLimit}|],
+  "",
+  [i|\# Finalization committee rewards:|],
```
Comment (td202) [inline src/Concordium/Client/Output.hs:1148]:
```suggestion
  [i|\# Finalization committee parameters:|],
```

File: src/Concordium/Client/Output.hs [hunk 2]
```diff
@@ -1160,6 +1166,9 @@ showRatio r =
       den = denominator r
    in show num ++ " / " ++ show den ++ " (approx " ++ show (realToFrac r :: Double) ++ ")"
 
+showPartsPerHundredThousands :: Types.PartsPerHundredThousands -> String
+showPartsPerHundredThousands (Types.PartsPerHundredThousands val) = show val ++ "/" ++ "100000"
```
Comment (td202) [inline src/Concordium/Client/Output.hs:1170]: I think it might be better to render this as a percentage.

**Review by td202 — APPROVED** [review]

File: src/Concordium/Client/Output.hs [hunk 3]
```diff
@@ -1160,6 +1166,9 @@ showRatio r =
       den = denominator r
    in show num ++ " / " ++ show den ++ " (approx " ++ show (realToFrac r :: Double) ++ ")"
 
+showPartsPerHundredThousands :: Types.PartsPerHundredThousands -> String
```
Comment (Bargsteen) [inline src/Concordium/Client/Output.hs:1169]: Please add docs describing the output format.

**Review by Bargsteen — CHANGES_REQUESTED** [review]
Looks fine apart from docs and Thomas' suggestion 😊

File: src/Concordium/Client/Output.hs (hunk 2 above)
Comment (abizjak) [reply src/Concordium/Client/Output.hs:1170]: I think just using the Show instance of PartsPerHundredThousands is also fine.

**Review by abizjak — APPROVED** [review]

File: src/Concordium/Client/Output.hs (hunk 3 above)
Comment (EmilLa1) [reply src/Concordium/Client/Output.hs]: Removed this. Using the `show` instance for `PartsPerHundredThousands`.

**Pull request merged**